src/main/java/com/hyperlocal/server/ShopController.java

```diff
@@ -259,54 +245,68 @@ public ShopController(PubSubTemplate pubSubTemplate) {
     JsonArray updateCommands = commands.getAsJsonArray("update");
     JsonArray deleteCommands = commands.getAsJsonArray("delete");
     CompletableFuture<QueryResult> statusPromise = connection.sendQuery("BEGIN");
-
-    // Process create commands
-    for (JsonElement createCommandRaw : createCommands) {
+    
+    if(createCommands.size() > 0) {
+      String[] insertPlaceholdersArray = new String[createCommands.size()];
+      Arrays.fill(insertPlaceholdersArray, Constants.INSERT_CATALOG_PLACEHOLDER);
+      String insertPlaceholders = String.join(", ", insertPlaceholdersArray);
+      String insertQuery = String.format(Constants.INSERT_CATALOG_STATEMENT, insertPlaceholders);
+      List<Object> insertQueryParameters = new ArrayList<Object>();
```

Comment:
Will it be good to use generatePlaceHolder function in Utilities.java here? It also does the same

Reply:
Cannot use `getPlaceHolderString()` here. This code is generating `(?, ?, ?, ?) ... {n times} ... (?, ?, ?, ?)`, not `?, ?, ... {n times} ... ?, ?". But yeah, thanks for the suggestions - I can use the utility function for delete commands' placeholders.